docs/static/monitoring/monitoring-output.asciidoc

```diff
@@ -20,28 +20,27 @@ Otherwise, monitoring data might be routed in different ways or to different pla
 [[logstash-monitoring-default]]
 ===== Default Configuration
 
-If a Logstash node does not explicitly define an {monitoring} output setting, 
+If a Logstash node does not explicitly define a monitoring output setting, 
 the following default configuration is used:
 
 [source,yaml]
 ---------------------------------------------------
-xpack.monitoring.elasticsearch.hosts: [ "http://localhost:9200" ]
+monitoring.elasticsearch.hosts: [ "http://localhost:9200" ]
 ---------------------------------------------------
 
-All data produced by {monitoring} for Logstash is indexed in the monitoring 
+All data produced by Logstash monitoring is indexed in the monitoring 
 cluster by using the `.monitoring-logstash` template, which is managed by the
 {ref}/es-monitoring-exporters.html[exporters] within {es}. 
 
 If you are working with a cluster that has {security-features} enabled, extra
 steps are necessary to properly configure Logstash. For more information, see 
 <<configuring-logstash>>. 
 
-IMPORTANT: When discussing security relative to the `elasticsearch` output, it
-is critical to remember that all users are managed on the production cluster, 
-which is identified in the `xpack.monitoring.elasticsearch.hosts` setting.
-This is particularly important to remember when you move from development 
-environments to production environments, where you often have dedicated 
-monitoring clusters.
+IMPORTANT: When discussing security relative to the `elasticsearch` output,
```

Comment:
I don't *think* this is still correct as we don't route through production clusters any more

Reply:
TBD